src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewService.java

```diff
@@ -179,9 +185,14 @@ private void handleFaceToFaceHearing(Adjournment adjournment, AdjournCaseTemplat
     private void setIntepreterDescriptionIfRequired(AdjournCaseTemplateBodyBuilder adjournCaseBuilder, SscsCaseData caseData) {
         if (isYes(caseData.getAdjournment().getInterpreterRequired())) {
             if (caseData.getAdjournment().getInterpreterLanguage() != null) {
-                adjournCaseBuilder.interpreterDescription(
-                    languageService.getInterpreterDescriptionForLanguageKey(
-                        caseData.getAdjournment().getInterpreterLanguage()));
+                String languageLabel = caseData.getAdjournment().getInterpreterLanguage().getValue().getLabel();
+                String languageKey = caseData.getAdjournment().getInterpreterLanguage().getValue().getCode();
+                Language language = signLanguagesService.getLanguageByHmcReference(languageKey);
+                String interpreterDescription = String.format("an interpreter in %s", languageLabel);
+                if (nonNull(language)) {
+                    interpreterDescription = String.format("a sign language interpreter (%s)", languageLabel);
+                }
```

Comment:
These can be reduced into a single String.format
```suggestion
                String interpreterDescription = String.format("a %sinterpreter (%s)", language != null ? "sign language " : "" , languageLabel);

```

Reply:
This will not produce exact result. This has a/an and then () for language lable in other scenario. So trying to keep it simple hence avoided ternary operator as well.